test/helpers/data/aggregated-data-helper.js

```diff
@@ -192,9 +192,33 @@ var addOffenderManager = function (inserts) {
       }
 
       var tiers = []
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 0, location: 'COMMUNITY'}))
       tiers.push(Object.assign({}, defaultTier, {tier_number: 1, location: 'COMMUNITY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 2, location: 'COMMUNITY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 3, location: 'COMMUNITY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 4, location: 'COMMUNITY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 5, location: 'COMMUNITY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 6, location: 'COMMUNITY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 7, location: 'COMMUNITY'}))
+
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 0, location: 'CUSTODY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 1, location: 'CUSTODY'}))
       tiers.push(Object.assign({}, defaultTier, {tier_number: 2, location: 'CUSTODY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 3, location: 'CUSTODY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 4, location: 'CUSTODY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 5, location: 'CUSTODY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 6, location: 'CUSTODY'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 7, location: 'CUSTODY'}))
+
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 0, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 1, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 2, location: 'LICENSE'}))
       tiers.push(Object.assign({}, defaultTier, {tier_number: 3, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 4, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 5, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 6, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 7, location: 'LICENSE'}))
```

Comment:
You can compress this into a forEach loop, look at how I've done it in the worker should be able to adapt that to here I think